public/css/app.css

```diff
@@ -1,5 +1,372 @@
-@import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,600,700');
+@import url('https://fonts.googleapis.com/css?family=Fira+Sans:300,400,500,600');
 
+html, body {
+  margin: 0 0 0 0;
+  padding:0 0 0 0;
+  color: #ffffff;
+  font-family: 'Fira Sans', sans-serif;
+  font-weight: 300;
+}
+
+body {
+  background: rgba(0, 62, 170,1);
+  background-image: linear-gradient(140deg, 
+    rgba(69, 161, 255,1),rgba(10, 132, 255,1),
+    rgba(0, 96, 223,1),rgba(0, 96, 223,1), rgba(0, 62, 170,1) );
+  background-repeat:no-repeat;
+  background-attachment: fixed;
+  background-position: center center;
+  background-size: cover;
+}
+
+h1, h2, h3, h4, h5 {
+  margin-bottom: 0;
+  font-family: 'Fira Sans', sans-serif !important;
+  font-weight:300;
+}
+
+
+.demi {
+  font-weight:400;
+}
+
+.medium {
+  font-weight:500;
+}
+
+.bold {
+  font-weight: 700;
```

Comment:
Should this be "600"?
I only see us importing the 300/400/500/600 font weights:

```css
@import url('https://fonts.googleapis.com/css?family=Fira+Sans:300,400,500,600');
```